CONTRIBUTING.md

```diff
@@ -1,55 +1,64 @@
-## Questions about Kodi?
+### Introduction
 
-To get your questions answered, please ask in the [Kodi community forums] (http://forum.kodi.tv/) or on IRC: #kodi on freenode.net
+**Kodi** uses github for development only, i.e. for *pull requests* and the review of such code.  
+**Do not open** an issue on github for your questions or bug reports.  
+**Do not comment** on a *pull request* unless you are involved in the testing of such or have something meaningful to contribute.  
+Not familiar with git? Start by looking at github's [collaborating pages](https://help.github.com/categories/collaborating/).
 
-Do not open an issue.
+#### Questions about Kodi?
 
-## Issue Reports
+To get your questions answered, please ask in the [Kodi community forum's](http://forum.kodi.tv/) or on **IRC:** [#kodi](http://webchat.freenode.net?nick=kodi-user&channels=%23kodi&prompt=1&uio=OT10cnVlde) on freenode.net
 
-Kodi uses github for development only, i.e. for pull requests and the discussion of code.
+#### Issue / bug reports and discussions
 
-So we use a hook script to automatically close new issue created by you.
+Issue/Bug reports are created and reviewed at [Kodi's bug tracker](http://trac.kodi.tv) using the Kodi forum's username and password.
 
-If you can, we encourage you to investigate the issue yourself and create a Pull Request for us to review.
+If you can, we encourage you to investigate the issue yourself and create a [pull request](https://help.github.com/articles/creating-a-pull-request/) for us to review.
 
-For bug reports, feature requests and all other support, please go to [Kodi community forums] (http://forum.kodi.tv/).
+For bug reports and related discussions, feature requests and all other support, please go to [Kodi community forum's](http://forum.kodi.tv/).
 
-## Pull Requests
+#### Pull Requests
 
-- Before creating a pull request please read our general code guidelines that can be found at
-  [Kodi development] (http://kodi.wiki/view/Development).
+- Before [creating a pull request](https://help.github.com/articles/creating-a-pull-request/) please read our general code guidelines that can be found at
```

Comment:
The link below says "coding guidelines" so I guess it should read the same here (not "code guidelines")